scorecard/tests/test_scorecard.py

```diff
@@ -134,3 +135,33 @@ def test_data_explorer_links(self):
         self.assertIn('http://portal:8002/table/cflow/?municipalities=BUF&year=2019&items=3010%2C3030%2C3040%2C3050%2C3060%2C3070%2C3100&amountType=AUDA', element)
         element = selenium.find_elements_by_css_selector(link_class)[19].get_attribute("href")
         self.assertIn('http://portal:8002/table/incexp/?municipalities=BUF&year=2019&items=0200%2C0400%2C1000&amountType=AUDA', element)
+
+    def test_grants_section(self):
+        selenium = self.selenium
+        selenium.get("%s%s" % (self.live_server_url, "/profiles/municipality-BUF-buffalo-city/"))
+
+        element = selenium.find_element_by_css_selector('#income-summary').text
+        self.assertIn('2018-2019 Audited actual', element)
+        self.assertIn('R6 040 712 322', element)
+
+        element = selenium.find_element_by_css_selector('#types-of-transfers').text
+        self.assertIn('2018-2019 Original budget', element)
+        self.assertIn('R183 861 100', element)
+        element = selenium.find_element_by_css_selector('#w-dropdown-list-11').get_attribute("innerHTML")
+        self.assertIn('2018-2019 Original budget', element)
+        self.assertIn('2019-2020 Original budget', element)
+
+        element = selenium.find_element_by_css_selector('#equitable-share').text
+        self.assertIn('2018-2019 Original budget', element)
+        self.assertIn('R79 021 100', element)
+        element = selenium.find_element_by_css_selector('#w-dropdown-list-12').get_attribute("innerHTML")
+        self.assertIn('2018-2019 Original budget', element)
+        self.assertIn('2019-2020 Original budget', element)
+
+        element = selenium.find_element_by_css_selector('#national-conditional-grants').text
+        self.assertIn('2019-2020 Allocations', element)
+        self.assertIn('R285 942 000', element)
+
+        element = selenium.find_element_by_css_selector('#provincial-transfers').text
+        self.assertIn('2019-2020 Original budget', element)
```

Comment:
how come this is the correct year, but in staging it's not?

Reply:
As we found out this was a result of the data not being provided - hopefully for a short time only